deprecated/NaviGator/xbox_joy_node/src/joy_node.cpp

```diff
@@ -1,11 +1,11 @@
 /*
```

Comment:
there is no reason for a diff in this file to exist. The files in deprecated should not be changed. They are only there for reference for future developers.

Reply:
Reverted changes to deprecated files.